urdu/Hamdard-Naunehal.md

```diff
@@ -0,0 +1 @@
+For online reading Click[here](https://archive.org/stream/HamdardNaunehalMarch2016HDPdfbooksfree.pk_201604/Hamdard_Naunehal_March_2016_HD_Pdfbooksfree.pk#page/n15/mode/2up)
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.